tests/all/cli_tests.rs

```diff
@@ -1853,6 +1853,41 @@ stderr [1] :: after empty
         run_wasmtime(&["run", "--argv0=foo.wasm", CLI_ARGV0, "foo.wasm"])?;
         Ok(())
     }
+
+    #[cfg(feature = "wasi-runtime-config")]
```

Comment:
Currently we assume that all features are enabled during testing, so it's ok to drop this and the marker below.